reference/opt/shaders-msl/frag/mrt-array.frag

```diff
@@ -29,7 +29,7 @@ Tx mod(Tx x, Ty y)
 fragment main0_out main0(main0_in in [[stage_in]])
 {
     main0_out out = {};
-    float4 FragColor[4];
+    float4 FragColor[4] = {};
```

Comment:
Any reason for this initializer?

Reply:
Adding an initializer was a convenient way of getting the variable to be declared early in `CompilerGLSL::emit_function()`.

It's not so important for output variables...but critical for input variables whose content will be filled by statements in `fixup_statements_in`, and need to be declared before those statements are emitted.

I could modify so that only input variables are initialized.

And I'm open to other mechanisms. I toyed with adding a new marker on variables to indicate that they needed to be declared early and then modify `CompilerGLSL::emit_function()` to check it...but in the end I figured adding an initializer could make use of the existing `CompilerGLSL::emit_function()` mechanism.

Reply:
I see. That's what I assumed. I was going to check if this was the case, but this saved me some time. It's fine as is.